src/transitions/3d-scroll-transition.js

```diff
@@ -60,6 +61,8 @@ export default class ScrollTransition3D extends React.Component {
           }`}
           style={this.getSlideStyles(index, positionValue)}
           key={index}
+          onClick={handleSelfFocus}
+          tabIndex={-1}
```

Comment:
Setting to `-1` lets us programmatically set the focus without having to set any specific tab index.